src/lingot-gui-mainframe.glade

```diff
@@ -178,181 +175,187 @@
           </packing>
         </child>
         <child>
-          <object class="GtkPaned" id="vertical_paned">
+          <!-- n-columns=3 n-rows=1 -->
```

Comment:
Here it is where it seems to be removing the `GtkPaned` introduced in 031e6191d0eeacaccc2f02f8d797e36b74997e55

Reply:
Should be fixed now. Tell me if you encounter any more errors. Resolve if it works